src/manual/task-execution/workers.md

```diff
@@ -0,0 +1,61 @@
+---
+filename: task-execution/workers.md
+title: Workers
+order: 20
+---
+
+Workers are the component responsible for actually executing tasks. Taskcluster
+operates on the "pull" model, where workers pull tasks from queues as they have
+available capacity.
+
+Typically workers are implemented as daemons running on compute instances in
+the cloud. As such, they have no REST API -- that is, there is no "worker
+service".
+
+## Worker Groups and Ids
+
+Each individual worker is identified by a `<workerGroup>/<workerId>` pair.
+This information can be useful to identify errant workers.
```

Comment:
I was thinking of the case where a worker starts misbehaving, and it's easy to see that all of the tasks with a particular weird error have the same workerId.  I reworded it a little (and used "misbehaving" instead of "errant")